src/checks/noDependencyProblems/noUnusedDependencies.ts

```diff
@@ -75,42 +75,43 @@ export default async function noUnusedAndMissingDependencies() {
       `);
     }
 
-    /**
-     * Temporary workaround to filter out @upleveled/eslint-config-upleveled peer dependencies
-     * not listed in `package.json`, which are flagged as missing dependencies by depcheck
-     *
-     * TODO: Remove this variable once this depcheck issue is fixed:
-     * https://github.com/depcheck/depcheck/issues/789
-     */
-    const missingDependenciesStdoutFiltered = missingDependenciesStdout
-      .split('\n')
-      .filter((missingDependency) => {
-        return !(
-          missingDependency.includes('./.eslintrc.cjs') &&
-          [
-            '@next/eslint-plugin-next',
-            '@typescript-eslint/eslint-plugin',
-            '@typescript-eslint/parser',
-            '@upleveled/eslint-plugin-upleveled',
-            'eslint-config-react-app',
-            'eslint-import-resolver-typescript',
-            'eslint-plugin-import',
-            'eslint-plugin-jsx-a11y',
-            'eslint-plugin-jsx-expressions',
-            'eslint-plugin-react-hooks',
-            'eslint-plugin-react',
-            'eslint-plugin-security',
-            'eslint-plugin-sonarjs',
-            'eslint-plugin-unicorn',
-          ].some((excludedDependency) =>
-            missingDependency.includes(excludedDependency),
-          )
-        );
-      })
-      .join('\n');
-
-    if (missingDependenciesStdoutFiltered) {
-      messages.push(`Missing dependencies found:
+    if (missingDependenciesStdout) {
+      /**
+       * Temporary workaround to filter out @upleveled/eslint-config-upleveled peer dependencies
+       * not listed in `package.json`, which are flagged as missing dependencies by depcheck
+       *
+       * TODO: Remove this variable once this depcheck issue is fixed:
+       * https://github.com/depcheck/depcheck/issues/789
+       */
+      const missingDependenciesStdoutFiltered = missingDependenciesStdout
+        .split('\n')
+        .filter((missingDependency) => {
+          return !(
+            missingDependency.includes('./.eslintrc.cjs') &&
+            [
+              '@next/eslint-plugin-next',
+              '@typescript-eslint/eslint-plugin',
+              '@typescript-eslint/parser',
+              '@upleveled/eslint-plugin-upleveled',
+              'eslint-config-react-app',
+              'eslint-import-resolver-typescript',
+              'eslint-plugin-import',
+              'eslint-plugin-jsx-a11y',
+              'eslint-plugin-jsx-expressions',
+              'eslint-plugin-react-hooks',
+              'eslint-plugin-react',
+              'eslint-plugin-security',
+              'eslint-plugin-sonarjs',
+              'eslint-plugin-unicorn',
+            ].some((excludedDependency) =>
+              missingDependency.includes(excludedDependency),
+            )
+          );
+        })
+        .join('\n');
+
+      missingDependenciesStdoutFiltered &&
```

Comment:
we don't usually do conditions like this

Reply:
[48177b8](https://github.com/upleveled/preflight/pull/378/commits/48177b8b5bb8fffc40a3874baefa366371f0e762)